src/util/pull-request-title.ts

```diff
@@ -16,59 +16,106 @@
 // at the script level are undefined, they are only defined inside function
 // or instance methods/properties.
 
-const DEFAULT_PATTERN = /^chore(\((?<branch>[\w-.]+)\))?: release ?(?<component>[\w-.]*)? v?(?<version>[0-9].*)$/;
+const DEFAULT_PR_TITLE_PATTERN =
+  'chore${scope}: release${component} ${version}';
```

Comment:
I'm a little worried that this loses some of the subtleties of the regex, such as the fact that 
```suggestion
const DEFAULT_PR_TITLE_PATTERN = 'chore${scope}: release${component} ${version}';
```

Reply:
I did not see the difference between this suggest commit to my commit. I have been using the `npm run fix` to format my commit.